src/index.b.js

```diff
@@ -0,0 +1,32 @@
+import './style.css';
+
+const taskContent = document.querySelector('.todo-header');
+
+const tasks = [];
+
+const displayTask = (task) => {
+  tasks.push(task);
+
+  const html = `
+    <div class="header">
+    <div class="check-content">
+      <input type="checkbox" class="checkbox" />
+      <div class="task"><p>${task}</p></div>
+    </div>
+    <i class="fa-solid fa-ellipsis-vertical"></i>
+  </div>`;
+  taskContent.insertAdjacentHTML('beforeend', html);
```

Comment:
- [ ] I could see that you didn't add index to your tasks here so I recommend to add indexes (i.e. if you're adding a 5th task to the list, the index of that task should equal to 5). as required and please update the index after deleting a task (i.e. if you're deleting the first task index 1 from the list, the index of the next task(2) should set to 1)..🔥✨💯✅

Reply:
Done. Thanks